tests/functional/desy/test_desy.py

```diff
@@ -113,6 +100,41 @@ def get_local_settings():
     }
 
 
+@pytest.fixture
+def get_local_settings_for_broken():
+    package_location = get_test_suite_path(
+        'desy',
+        'fixtures',
+        'ftp_server',
+        'DESY',
+        'broken',
+        test_suite='functional',
+    )
```

Comment:
I'm not sure if it helps here, but if you did not check it out you might want to look into the pytest temporary dirs helpers: https://docs.pytest.org/en/latest/tmpdir.html#base-temporary-directory

Reply:
it does and it's super easy to use: see e.g. https://github.com/inspirehep/inspire-next/blob/7261d3723875e7e093965912183629e5b5beb6ef/tests/unit/refextract/test_refextract_utils.py

Reply:
:+1: just forgot :dancer:

Reply:
I've tried to use it here, but I think it's not the case. `tmpdir` uses its own `base_tmp_dir` in which subfolders are created. Here we want a file inside the test folder because that one is the one that will be crawled during the test.